src/content/ko/fundamentals/performance/prpl-pattern/index.md

```diff
@@ -0,0 +1,197 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2016-09-28 #}
+{# wf_published_on: 2016-09-28 #}
+
+# The PRPL Pattern {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+개요 : PRPL 은 우리가 느끼기에 굉장한 잠재력을 지닌 새로운 패턴입니다. 이번 단계에서는 이 패턴에 대한 아이디어를 토대로 반복적인 실험을 해보고 매력적인 장점을 발견할 수 있도록 더 많은 정보를 얻으려고 합니다.
+
+모바일 웹은 너무 느립니다. 지난 시간 동안 웹은 다큐멘트 중심의 플랫폼에서 일급 함수 어플리케이션 플랫폼으로 발전해왔습니다.
+플랫폼과 도구 측면에서 [서비스 워커](/web/fundamentals/getting-started/primers/service-workers) 와 같은 기술적인 진보와, 앱을 구현할 때 사용하는 기술들 덕분에 사용자들은 네이티브 앱에서 할 수 있던 것들을 웹에서도 할 수 있게 되었습니다.
+
+이와 동시에 대다수의 컴퓨팅 환경이 빠르고 안정적인 네트워크 연결을 제공하는 강력한 데스크톱에서,
+상대적으로 느리거나, 품질이 낮은 연결을 제공하는 모바일 장치들로 옮겨왔습니다.
+몇십억 명의 사용자들이 온라인을 경험하게 되는 세계에서는 이러한 주장이 사실로 여겨지고 있습니다.
+
+불행하게도 우리가 데스크톱에서 강력하고 기능이 많은 웹 앱을 만들고 배포하기 위해 고안했었던 패턴들이
+이제 일반적으로는 모바일 기기에서 너무 시간이 오래 걸리게 되었습니다. 이로 인해 많은 사용자들이 사이트가 느리면 그냥 포기하게 되죠.
+
+이러한 상황은 모바일 웹 경험을 더 빠르게 제공할 수 있고, 최신 웹 플랫폼 특징들을 녹여낼 수 있는 새로운 패턴들을 제작하는 계기가 됩니다.
+그리고 PRPL 은 그 중 하나가 되겠죠.
+
+## PRPL 패턴
+
+PRPL 는 프로그레시브 웹 앱 (PWA) 를 구조화 하고 동작시킬 수 있는 패턴입니다. 앱 시작과 배포에서 성능의 강점을 가지죠.
+PRPL 의 각 철자는 아래의 의미를 갖습니다.
+
+*  **푸시(Push)** : 초기 URL 접근에 대한 크리티컬 리소스 푸시
```

Comment:
oh, that is what I wanted to say. I thought you translate them without origin word. I'll try to review tranlations from the next time. :)